### PR TITLE
[DOCS-8839] Add OP Q3 Live Capture doc

### PR DESCRIPTION
**Pull request by maycmlee**
### What does this PR do? What is the motivation?

- Adds the Live Capture doc for the OP 2.2 release.

[DOCS-8839](https://datadoghq.atlassian.net/browse/DOCS-8839)

### Merge instructions

- [ ] Please merge after reviewing

### Additional notes

[DOCS-8839]: https://datadoghq.atlassian.net/browse/DOCS-8839?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: content/en/observability_pipelines/live_capture.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  text: "Set up pipelines"
+---
+
+{{< beta-callout url="" header="false" btn_hidden="true">}}
+Live Capture is in Preview.
+{{< /beta-callout >}}
+
+## Overview
+
+Use Live Capture to see the data received from and sent out of each source or processor, specifically the following information:
```
Comment (maycmlee) [inline content/en/observability_pipelines/live_capture.md:16]:
```suggestion
Use Live Capture to view data received from and sent out of each source or processor. 
Specifically, the following information is shown:
```

File: content/en/observability_pipelines/live_capture.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+## Capture events
+
+1. Navigate to [Observability Pipelines][1].
+1. Select your pipeline.
+1. Click the cog for the source or processor that you want to capture events for.
+1. Select **Capture and view events** in the side panel.
+1. Click **Capture**.
+1. Click **Confirm** to start capturing events.<br>**Note**: Capturing events usually takes up to 60 seconds. Captured data is visible to all users with view access, and is stored in the Datadog Platform for 72 hours.
+1. After the capture is complete, click a specific capture event to see data that was received and the data that was sent out. You can also search for specific events in the search bar. Use the dropdown menu next to the search bar to show events based on status (`MODIFIED`, `UNMODIFIED`, `DROPPED`, and `REDUCED`).
```
Comment (maycmlee) [inline content/en/observability_pipelines/live_capture.md:33]:
```suggestion
1. After the capture is complete, click a specific capture event to see the data that was received and sent out. You can also search for specific events in the search bar. Use the dropdown menu next to the search bar to show events based on status (`MODIFIED`, `UNMODIFIED`, `DROPPED`, and `REDUCED`).
```

**Review by aliciascott — APPROVED** [review]
looks good! Just a question is this intended to not be displayed on the Left hand menu Nav since it's in Preview?

**Comment by maycmlee** [comment]
> looks good! Just a question is this intended to not be displayed on the Left hand menu Nav since it's in Preview?

I was waiting for confirmation of the feature name :D. Thanks for the reminder though! I think I'm just going to add it without the confirmation.

**Pull request merged**